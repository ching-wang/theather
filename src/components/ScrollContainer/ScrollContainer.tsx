import React from "react";
import { Wrapper } from "./scrollContainer.styles";

export const ScrollContainer = (props: {children: React.ReactNode}) => {
    return (
        <Wrapper>
            {props.children}
        </Wrapper>
    )
}

export default ScrollContainer;