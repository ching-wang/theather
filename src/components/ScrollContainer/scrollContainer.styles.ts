import styled from "styled-components";


export const Wrapper = styled.div`
  overflow-y: auto;
  height: 800px;
  padding-top: 110px;
  display: flex;
  justify-content: center;
`