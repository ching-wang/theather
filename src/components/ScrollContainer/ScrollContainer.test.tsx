import React from "react";
import ScrollContainer from "./ScrollContainer";
import renderer from "react-test-renderer";

describe(ScrollContainer.name, () => {
    it("renders", () => {
        const tree = renderer
            .create(
                <ScrollContainer>
                    <div>
                        <h1>
                            Hello, world!
                        </h1>
                    </div>
                    <div>
                        <h2>Hi!</h2>
                    </div>
                </ScrollContainer>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
