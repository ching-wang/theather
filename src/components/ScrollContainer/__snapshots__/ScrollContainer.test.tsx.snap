// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollContainer renders 1`] = `
<div
  className="sc-bdfBQB goRYlO"
>
  <div>
    <h1>
      Hello, world!
    </h1>
  </div>
  <div>
    <h2>
      Hi!
    </h2>
  </div>
</div>
`;
